hack/vendor.sh

```diff
@@ -44,7 +44,9 @@ clone git github.com/boltdb/bolt v1.1.0
 clone git github.com/docker/distribution 568bf038af6d65b376165d02886b1c7fcaef1f61
 clone git github.com/vbatts/tar-split v0.9.11
 
-clone git github.com/docker/notary 45de2828b5e0083bfb4e9a5a781eddb05e2ef9d0
+# get desired notary commit, might also need to be updated in Dockerfile
+clone git github.com/docker/notary ffca6fb522fc0dc9d582c848b20746e9cda9ca87
```

Comment:
@icecrime Do you mean tag it with the docker commit, the docker version, or both?  (asking in case we have to re-vender notary twice in the same docker version - then two commits would be tagged with the same docker version)